src/character.cpp

```diff
@@ -2347,6 +2347,10 @@ void Character::practice( const skill_id &id, int amount, int cap, bool suppress
         amount *= 2;
     }
 
+    if( has_trait_flag( "PACK_HUNTER" ) && skill.is_combat_skill() ) {
+        amount *= 2;
+    }
+
```

Comment:
This flag should not be used here. Either add the `PRED3` flag to your mutation, or add one for what it does in common with this.

Reply:
Reverted all changes to the file, as they were not needed after all.